tests/sentry/web/frontend/test_js_sdk_dynamic_loader.py

```diff
@@ -1,10 +1,35 @@
+from functools import cached_property
+
+import pytest
 from django.conf import settings
 from django.urls import reverse
 
 from sentry.testutils import TestCase
 
 
 class JavaScriptSdkLoaderTest(TestCase):
+    @pytest.fixture(autouse=True)
+    def set_settings(self):
+        settings.JS_SDK_LOADER_SDK_VERSION = "0.5.2"
+        settings.JS_SDK_LOADER_DEFAULT_SDK_URL = "https://browser.sentry-cdn.com/%s/bundle%s.min.js"
+
+    @cached_property
+    def path(self):
+        return reverse("sentry-js-sdk-dynamic-loader", args=[self.projectkey.public_key])
+
+    def test_noop_no_pub_key(self):
+        resp = self.client.get(reverse("sentry-js-sdk-dynamic-loader", args=["abc"]))
+        assert resp.status_code == 200
+        self.assertTemplateUsed(resp, "sentry/js-sdk-loader-noop.js.tmpl")
+
+    def test_noop(self):
+        settings.JS_SDK_LOADER_DEFAULT_SDK_URL = ""
+        resp = self.client.get(
+            reverse("sentry-js-sdk-dynamic-loader", args=[self.projectkey.public_key])
```

Comment:
Is this ok to become a region specific endpoint in the the future? With only a project key parameter, we'd have to eventually use URLs like `https://us.sentry.io/api/0/.*` style URLs.

Reply:
Yes this should be fine since this case is only there for self-hosted users. In the production `getsentry` case, we are using a Fastly CDN (hence the configurable URL).